### DISCUSSION TITLE
L'Or rouge

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#33] [Q&A]
Commentaires sur ce quiz

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/artificialOxygen/index.html
- Dépôt Github (privé pour le moment): https://github.com/ArtificialOxygen/quiz-tim
- Github.io :

**Comment by ArtificialOxygen** [comment]
Avis pour les personnes qui voudront faire un retour sur mon quiz :smile:

Je voulais juste vous faire part de ce qui est incomplet :

- Faire un état désactivé sur les boutons radios lorsqu'on affiche le résultat d'une réponse et un état différent lorsque le bon ou le mauvais bouton radio a été coché. Si on se trompe, il n'y a pas de moyen "évident" de savoir qu'elle était la bonne réponse mise à part lire le texte explicatif.

- Je n'ai pas ajouté de bouton pour recommencer le quiz, puisque je l'ai totalement oublié. J'ai simplement invité l'utilisateur à aller faire d'autres tests en cliquant sur la bannière d'accueil.

**Comment by evefevrier** [comment]
@ArtificialOxygen 
Ce quiz est éblouissant! 🥇 
Tant qu'à faire la promo de Héma-Québec, pourquoi ne pas ajouter l'hyperlien dans la page de résultat?

**Reply by ArtificialOxygen** [reply]
Justement, je voulais remplacer le SVG des deux gouttes de sang par le logo d'Héma Québec qui, lorsqu'on clique dessus, nous aurait amené à leur site :smile:, mais faute de temps, je ne l'ai pas fait.
